requirements.txt

```diff
@@ -4,6 +4,8 @@ gpiozero
 picamera
 sounddevice
 sanic
+gettext
```

Comment:
gettext and locale is builtin with python3. don't need to require gettext and locale, they can't be installed individually